ramp-database/ramp_database/testing.py

```diff
@@ -123,7 +123,8 @@ def _delete_line_from_file(f_name, line_to_delete):
         f.truncate()
 
 
-def setup_ramp_kit_ramp_data(ramp_config, problem_name, force=False):
+def setup_ramp_kit_ramp_data(ramp_config, problem_name, force=False,
+                            check_notebook=False):
```

Comment:
Maybe,
```suggestion
                            convert_notebook=True):
```
and then explicitly set to to false for unit tests? For event deployment I agree that it's nice to have an HTML notebook. Notebook preview on github mostly doesn't work.